docs/advanced-topics/legacy/wf-legacy-registration.rst

```diff
@@ -0,0 +1,88 @@
+Legacy Workflow Registration Methods
```

Comment:
Filename, I would spell out workflow (instead of "wf"). It appears in the URL, and we generally don't abbreviate